app/src/app/components/view-profile/view-profile.component.html

```diff
@@ -157,28 +171,38 @@ <h4 class="app-profile-general-dropdown-heading">{{'APP.EDIT_PROFILE.ADDRESS' |
           <input type="text" value="" *ngIf="!contactSectionButtonsToggle" formControlName="companyAddress"
             class="app-profile-general-dropdown-input-details app-selected">
         </div>
-        <mat-divider class="app-general-dropdown-divider" ></mat-divider>
-        <div class="app-general-dropdown-details" >
+        <mat-divider class="app-general-dropdown-divider"></mat-divider>
+        <div class="app-general-dropdown-details">
           <h4 class="app-profile-general-dropdown-heading">{{'APP.EDIT_PROFILE.POST_CODE' | translate}}</h4>
           <p class="app-profile-general-dropdown-details" *ngIf="contactSectionButtonsToggle">{{personDetails?.company?.postcode}}</p>
           <input type="text" value="" *ngIf="!contactSectionButtonsToggle" formControlName="companyPostCode"
             class="app-profile-general-dropdown-input-details app-selected">
-
         </div>
-        <mat-divider class="app-general-dropdown-divider" ></mat-divider>
-        <div class="app-general-dropdown-details" >
+        <mat-divider class="app-general-dropdown-divider"></mat-divider>
+        <div class="app-general-dropdown-details">
           <h4 class="app-profile-general-dropdown-heading">{{'APP.EDIT_PROFILE.TELEPHONE' | translate}}</h4>
           <p class="app-profile-general-dropdown-details" *ngIf="contactSectionButtonsToggle">{{personDetails?.company?.telephone}}</p>
           <input type="tel" value="" *ngIf="!contactSectionButtonsToggle" formControlName="companyTelephone"
             class="app-profile-general-dropdown-input-details app-selected">
-
+            <div *ngIf="profileForm.get('companyTelephone').invalid &&
```

Comment:
not sure if that fully applies, but can you try to follow the material input example for the validation and use "mat-error" ? https://material.angular.io/components/form-field/overview#error-messages
example:
```
<mat-error *ngIf="email.invalid">{{getErrorMessage()}}</mat-error>
```